.travis.yml

```diff
@@ -16,6 +32,12 @@ notifications:
       - https://webhooks.gitter.im/e/d94969c3b01b22cbdcb7
     on_success: change
     on_failure: change
-    on_start: false
-matrix:
-  fast_finish: true
+    on_start: never
+deploy:
+  provider: bintray
+  file: "build/bintray_deploy.json"
+  user: "ctrlaltca"
+  key:
+    secure: b9Nv8FeYJGaTs1bZD8z0a7VksmEDkgVSDu8Os1ldYVaE6AwgYTt++RAbX60eu6jRb0G1bUCTWNF5wh3vhjYNl2xrj7N89O5FQAIgQcyWveAU91pn2I92pMcRfhHMIUVLwgzHoSy9JQPFG3ecE6CtWAi9+rbqcRwQRg+A2jfd7yI=
```

Comment:
Yes, this has been "encrypted" by travis and is supposed to be decrypted correctly only when used from the correct repo. This is currently working for my fork cockatrice/CtrlAltCa, you'll need to regenerate this for the Cockatrice/Cockatrice repo. (http://docs.travis-ci.com/user/encryption-keys/ )